libs/category/src/resolvers/category-page/category-page.resolver.ts

```diff
@@ -28,7 +30,7 @@ export class DaffCategoryPageResolver implements Resolve<Observable<boolean>> {
       id: route.paramMap.get('id'), page_size: this.defaultCategoryPageSize
 		}));
 
-		return this.dispatcher.pipe(
+		return isPlatformBrowser(this.platformId) ? of(true) : this.dispatcher.pipe(
```

Comment:
Could you make the stream around `isPlatformBrowser()` e.g. `of(isPlatformBrowser(this.platformId).pipe()`

Reply:
It would be helpful if you explained why it would be better that way, unless it is obvious. It isn't obvious to me why it would be better this way:
```
return of(isPlatformBrowser(this.platformId)).pipe(
	switchMap((isBrowser) => isBrowser ? of(isBrowser) : this.dispatcher.pipe(
		ofType(DaffCategoryActionTypes.CategoryLoadSuccessAction, DaffCategoryActionTypes.CategoryLoadFailureAction),
		mapTo(true),
		take(1)
	))
);
```

Reply:
It's not particularly better, it's more a stylistic desire to keep everything inside the pipe. It doesn't matter either way.